Source/ACE.Server/WorldObjects/Entity/CreatureVital.cs

```diff
@@ -76,7 +76,7 @@ public uint Base
         {
             get
             {
-                var attr = AttributeFormula.GetFormula(creature, Vital);
+                var attr = AttributeFormula.GetFormula(creature, Vital, false);
```

Comment:
hm that looks reasonable for Base maybe, but it breaks Current/MaxValue
dunno if you are familiar with the /showstats command, but it is very useful for testing these scenarios
MaxValue would also need to be changed to this:
```
        public uint MaxValue
        {
            get
            {
                var attr = AttributeFormula.GetFormula(creature, Vital, true);

                uint total = StartingValue + Ranks + attr;
```

Reply:
makes sense, thanks for reviewing